libr/io/p/io_r2k.c

```diff
@@ -238,10 +239,91 @@ static int Init (const char * driverPath) {
 }
 #endif
 
+#if __linux__
+#include <sys/ioctl.h>
+
+struct r2k_data {
+	int pid;
+	unsigned long addr;
```

Comment:
use ut64 here, userland doesnt knows if its talking to a 32 or 64bit kernel, so that must be casted down in kernel land